api/src/main/java/org/eclipse/microprofile/faulttolerance/Fallback.java

```diff
@@ -44,4 +44,10 @@
      * @return the fallback class
      */
     Class<? extends FallbackHandler<?>> value();
+    /**
+     * Specify the method name to be fallbacked to. This method belongs
+     * to the same class as the method to fallback.
+     * @return the local method to fallback to
+     */
+    String fallbackMethod() default "";
```

Comment:
Its confusing to use both `FallbackHandler` and `fallbackMethod` as the usage would imply to use that method on the handler implementation.

While I see it demonstrated quite often to point to another method on the same class, I don't see any real world scenarios where this would be the case.  I would prefer to use these two in conjunction:

- if FallbackHandler is specified, then use `fallbackMethod` on the handler to execute the fallback
- If no handler specified, then its assumed to be this class
- If handler is specified but no method name specified, then assume there is a method on the handler with the same name, args and return type.